### PR TITLE
Materialization Loading updates

### PR DESCRIPTION
**Pull request by Georgehe4**

### CODE REVIEW

File: mango-core/src/main/scala/org/bdgenomics/mango/models/AlignmentRecordMaterialization.scala [hunk 1]
```diff
@@ -161,7 +161,8 @@ object AlignmentRecordMaterialization {
   def loadFromBam(sc: SparkContext, region: ReferenceRegion, fp: String): AlignmentRecordRDD = {
     AlignmentTimers.loadBAMData.time {
       val idxFile: File = new File(fp + ".bai")
-      if (idxFile.exists()) {
+      val idxFile2: File = new File(fp.dropRight(4) + ".bai")
+      if (idxFile.exists() || idxFile2.exists) {
```
Comment (akmorrow13) [inline mango-core/src/main/scala/org/bdgenomics/mango/models/AlignmentRecordMaterialization.scala]: rm idxFile2

Comment (akmorrow13) [inline mango-core/src/main/scala/org/bdgenomics/mango/models/AlignmentRecordMaterialization.scala]: I would rather just call loadIndexedBam() without a check for bai and handle the exception if no file is found

**Comment by akmorrow13** [comment]
One commit fix, other than that looks good @Georgehe4 . Thanks!

**Comment by Georgehe4** [comment]
Moved to #254

**Pull request closed without merging**